docs/marionette.collectionviewadvanced.md

```diff
@@ -419,3 +421,32 @@ var MyCollectionView = Mn.CollectionView.extend({
   }
 });
 ```
+
+## Binding `ui`
+
+By default, `CollectionView` will not bind anything defined in `ui`. As it has
+no direct `template` of its own to manage, this isn't a particular issue.
+There may be instances where you want to bind the `ui` element to listen to
+children on the collection directly. For example, to bind `triggers` or
```

Comment:
oddly enough `triggers` works without binding.  This is because it uses the text selectors to do the `@ui` stuff regardless if the ui has been bound.

https://jsfiddle.net/784y75ks/1/

This may only be for the `ui` hash and `getUI`